config/babel-preserve-modules.config.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+  presets: [
+    [
+      '@babel/preset-env',
+      {
+        modules: false,
+      },
+    ],
+    '@babel/preset-react',
+  ],
+  plugins: [
+    '@babel/plugin-proposal-object-rest-spread',
+    '@babel/plugin-proposal-class-properties',
+    '@babel/plugin-syntax-dynamic-import',
+  ],
+  env: {
+    i18n: {
+      plugins: [
+        [
+          'react-intl',
+          {
+            messagesDir: './temp/babel-plugin-react-intl',
+            moduleSourceName: '@edx/frontend-i18n',
+          },
+        ],
+      ],
+    },
+    test: {
+      presets: [
+        '@babel/preset-env',
```

Comment:
Don't you need the react preset here too?  Or does this get overlaid on top of the normal presets?

Reply:
it gets overlaid

Reply:
https://babeljs.io/docs/en/options#env